jormungandr/src/network/subscription.rs

```diff
@@ -259,7 +274,7 @@ impl Sink<net_data::Fragment> for FragmentProcessor {
 
     fn poll_ready(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Error>> {
         if self.buffered_fragments.len() >= buffer_sizes::inbound::FRAGMENTS {
-            ready!(self.poll_send_fragments(cx));
+            ready!(self.poll_send_fragments(cx))?;
```

Comment:
I'm not sure about this. Since we were not checking them before I did not know if it was the expected behavior. I can `map_err` as the others and just `debug!` the error there instead of lifting.

Reply:
Good catch, the sending errors are fatal and must be propagated.